external/dhcpd/dhcpd_lwip.c

```diff
@@ -129,13 +129,15 @@ static int _recv_mqmsg(mqd_t mq, char *msg, size_t msglen, int *prio)
 			if (errno == EAGAIN && errno == EINTR) {
 				continue;
 			}
-			ndbg("dhcps _recv_mqmsg error %d\n", errno);
+			ndbg("dhcps error %d\n", errno);
```

Comment:
How about like below? Then we can reduce the time.
Because `errno` tries to read the error number from tcb on every time.
```
int err_no = get_errno();
if (err_no == EAGAIN && err_no == EINTR) {
    continue;
}
ndbg("xxx %d\n", err_no);
```